mvc/controllers/areas.rst

```diff
@@ -1,8 +1,93 @@
-.. include:: /../common/stub-topic.txt
+Areas
+=====
+By `Tom Archer`
 
-|stub-icon| Areas
-===========================================
+:doc:`Areas </controllers/areas>` provide a way to separate a large MVC application into semantically-related groups of models, views, and controllers. Let's take a look at an example to illustrate how Areas are created and used. Let's say you have a store app that has two distinct groupings of controllers and views: Products and Services.
 
-.. include:: /../common/stub-notice.txt
+Instead of having all of the controllers housed under the Controllers parent directory, and all the views housed under the Views parent directory, you could use Areas to group your views and controllers according to the area (or logical grouping) with which they're associated.
 
-.. _issue: https://github.com/aspnet/Docs/issues/121
+- Project name
+
+  - Areas
+
+    - Products
+
+      - Controllers
+
+        - HomeController
+
+      - Views
+
+        - Home
+
+          - Index.cshtml
+
+    - Services
+
+      - Controllers
+
+        - HomeController
+
+      - Views
+
+        - Home
+
+          - Index.cshtml
+
+Once you've defined the folder hierarchy, you need to tell MVC that each controller is associated with an area. You do that by decorating the controller name with the ``[Area]`` attribute.
+
```

Comment:
Call out that the views are resolved using the file path? i.e., folder naming is significant. You need an "Areas" folder etc. `[area]/Views/[controller]/[action].cshtml` ... The example shows what to do, but it's not immediately clear to me that names are significant.